ee/clickhouse/models/team.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from typing import List
+
+from ee.clickhouse.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+logger = logging.getLogger(__name__)
+
+# Note: Session recording, dead letter queue, logs deletion will be handled by TTL
+TABLES_TO_DELETE_FROM = [
+    "events",
+    "person",
+    "person_distinct_id",
+    "person_distinct_id2",
+    "groups",
+    "cohortpeople",
+    "person_static_cohort",
+]
+
+
+def delete_teams_data(team_ids: List[int]):
```

Comment:
non-blocking

I assume these alter table ops are going to be reasonably quick and we don't need to worry about e.g. celery timeouts?

Reply:
They will return immediately as this operation is implemented as an async mutation. https://clickhouse.com/docs/en/sql-reference/statements/alter/delete/